contracts/finality/src/contract.rs

```diff
@@ -146,6 +150,7 @@ pub(crate) mod tests {
             admin: init_admin.to_string(),
             consumer_id,
             is_enabled: true,
+            min_pub_rand: 1,
```

Comment:
For tests I think we can pick a bigger value. There is already validation that pub rand commit should contain >=1 pub rand values